src/formio/templates/checkbox.js

```diff
@@ -1,21 +1,19 @@
-import { applyPrefix } from '../utils';
-
 const TEMPLATE = `
-<div class="${applyPrefix('checkbox')}">
+<div class="utrecht-form-field utrecht-form-field--checkbox utrecht-form-field--distanced">
     <{{ctx.input.type}}
       ref="input"
       {% for (var attr in ctx.input.attr) { %}
       {{attr}}="{{ctx.input.attr[attr]}}"
       {% } %}
       id="{{ctx.instance.id}}-{{ctx.component.key}}"
       {% if (ctx.checked) { %}checked=true{% } %}
+      class="utrecht-form-field__input utrecht-checkbox"
     >
-
         {{ctx.input.content}}
     </{{ctx.input.type}}>
 
-    <div class="${applyPrefix('checkbox__checkmark')}"></div>
-    <label class="${applyPrefix('checkbox__label')} {{ctx.input.labelClass}}" for="{{ctx.instance.id}}-{{ctx.component.key}}">
+<!--    <div class="checkbox__checkmark"></div>-->
```

Comment:
Not a big fan of leaving in commented out pieces of HTML once past the active development phase